windowsforms/Sudoku/VB/ReadMe.MD

```diff
@@ -0,0 +1,42 @@
+languages:
+    - vb
+products:
+    - dotnet-core
+    - windows
+page_type: sample
+name: ".NET Core WinForms Sudoku"
+urlFragment: "Sudoku-vb"
+description: "Sudoku is a number puzzle game, played on a nine by nine cell grid"
+# Sudoku
+## Requires
+- Visual Studio 2019 Version 16.5.0 Preview 1.0 or later
+- .Net Core 3.1
+
+## License
+- MIT
+
+## Technologies
+  - Windows Forms
+  - dotnet-core
+
+## Topics
+- Puzzle Game
+
+## Updated
+- 12/12/2019
+
```

Comment:
I don't see those are useful much.

Reply:
I copied it from archived file. I am fine deleting it.

Reply:
I think it is fixed